tests/test_copy_command.py

```diff
@@ -113,6 +135,20 @@ def test_invalid_format():
         )
 
 
+def test_fixed_width_format_without_widths():
+    copy = dialect.CopyCommand(
+        tbl,
+        format=commands.Format.fixed_width,
+        data_location='s3://bucket',
+        access_key_id=access_key_id,
+        secret_access_key=secret_access_key
+    )
+    with pytest.raises(sa_exc.CompileError) as errinfo:
+        compile_query(copy)
+
+    assert str(errinfo.value).startswith("'fixed_width' argument required")
```

Comment:
I think pytest raises provides a shortcut for this

Reply:
I didn't see it in the documentation for some reason.

Reply:
Fixed!